### PR TITLE
example for speed added

### PR DESCRIPTION
**Pull request by hackertron**
This PR is regarding this issue : https://github.com/processing/p5.js/issues/1954

example for speed added . 

Please review it !!

thanks !!

### CODE REVIEW

**Comment by Spongman** [comment]
looks good. a couple of things:
- the docs samples usually use a canvas size of 100x100. if your UI can't fit in that space, you might want to set `<div class="norender">` to prevent it appearing on the reference page.
- you should fix the indentation (2 spaces, no tabs).

**Comment by hackertron** [comment]
okay will fix it 👍

**Comment by hackertron** [comment]
Check it again !!

File: lib/addons/p5.dom.js [hunk 1]
```diff
@@ -2158,6 +2158,69 @@
...
+   *  }
+   * // play song in reverse
+   * function reverse_speed() {
+   *  ele.play();
+   *  ele.speed(-1);
+   *  }
+   * // stop song play
+   * function stop_song() {
+   *  ele.stop();
+   *  }
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: can you add one space in the line above, and remove the space here

File: lib/addons/p5.dom.js [hunk 2]
```diff
@@ -2158,6 +2158,69 @@
...
+   * button.mousePressed(stop_song);
+   *
+   * button = createButton('PLAY!');
+   * button.position(500, 68);
+   * button.mousePressed(play_speed);
+   *
+   * }
+   * // play song at 2x speed
+   * function twice_speed() {
+   *  ele.play();
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: these lines should be indented 2 spaces

File: lib/addons/p5.dom.js [hunk 3]
```diff
@@ -2158,6 +2158,69 @@
...
+   * //this example shows playback speed
+   * //twice , half and reverse playback
+   * 
+   * //We will store the p5.MediaElement
+   * //object in here
+   * var ele;
+   * function setup() {
+   * 
+   * createCanvas(710, 400);
+   * //Here we create a p5.MediaElement object
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: these lines should be indented 2 spaces

File: lib/addons/p5.dom.js [hunk 4]
```diff
@@ -2158,6 +2158,69 @@
...
+   *  }
+   * // stop song play
+   * function stop_song() {
+   *  ele.stop();
+   *  }
+   *
+   * // play song normally
+   * function play_speed() {
+   *  ele.play();
+   *  }
```
Comment (lmccart) [inline lib/addons/p5.dom.js]: can you add one space in the line above, and remove the space here

File: lib/addons/p5.dom.js (hunk 2 above)
Comment (lmccart) [inline lib/addons/p5.dom.js]: could we just call ele.loop() in setup, and then just the ele.speed() lines in these functions?

**Pull request closed without merging**